optimum/exporters/tasks.py

```diff
@@ -1093,7 +1099,11 @@ def _validate_framework_choice(framework: str):
 
     @staticmethod
     def get_model_class_for_task(
-        task: str, framework: str = "pt", model_type: Optional[str] = None, model_class_name: Optional[str] = None
+        task: str,
+        framework: str = "pt",
+        model_type: Optional[str] = None,
+        model_class_name: Optional[str] = None,
+        library: Optional[str] = "transformers",
```

Comment:
Why Optional? The code below will fail if library is None

Reply:
True, removed